README.md

```diff
@@ -1,2 +1,74 @@
 # packmap
+
 Turn a package.json into a browser-ready import map
+
+## Quickstart
+
+Install the library:
+
+```sh
+npm install --save-dev packmap
+```
+
+Now add the following to your package.json:
+
+```json
+{
+  "scripts": {
+    "build": "packmap"
+  }
+}
+```
+
+And run the following:
+
+```sh
+npm run build
+```
+
+## Explanation of packmap
+
+Packmap will create a browser-ready directory and import map from a package.json file and node_modules. To use it, your packages must meet the following standards:
+
+1. All package.json `dependencies` must be in-browser dependencies instead of build-time dependencies.
```

Comment:
How do I tell whether a dependency is an in-browser dependency or a build-time dependency? (I suspect a lot of folks getting started with this will have this question and it might be worth adding to the README?)

Reply:
All dependencies on openmrs-esm packages are in-browser. Additionally, `react`, `rxjs`, and `react-dom` are in-browser. You can tell if it's an in-browser dependency by looking at you [webpack externals](https://webpack.js.org/configuration/externals/). See https://github.com/openmrs/openmrs-esm-login/blob/master/webpack.config.js#L47 as an example

I've been going back and forth on whether to make packmap OpenMRS-specific or general-purpose. I've been leaning towards general purpose, which is why I created documentation in a github Readme instead of OpenMRS wiki. So I think the OpenMRS wiki will have additional documentation for things that are openmrs-specific, including the answer to your question.

Reply:
Nice, thanks!